src/Package.appxmanifest

```diff
@@ -81,7 +81,7 @@
           </com:ComServer>
         </com:Extension>
         <uap3:Extension Category="windows.appExtension">
-          <uap3:AppExtension Name="com.microsoft.devhome" Id="PG-SP-ID1" PublicFolder="Public" DisplayName="ms-resource:AppDisplayNameCoreExt" Description="ms-resource:AppDescriptionCoreExt">

```

Comment:
These DisplayNames should be reverted? Looks like Build.ps1 leaves this in a dirty state.

Reply:
Thanks, I'll get these reverted.  I'll also take a look to see if we can fix build.ps1 in a separate PR, since this isn't the first time I've had these inadvertently get included.